mesonbuild/mformat.py

```diff
@@ -839,7 +839,13 @@ def load_configuration(self, configuration_file: T.Optional[Path]) -> FormatterC
 
             for f in fields(config):
                 getter = f.metadata['getter']
-                value = getter(cp, cp.default_section, f.name, fallback=None)
+                try:
+                    value = getter(cp, cp.default_section, f.name, fallback=None)
+                except ValueError as e:
+                    value = str(e).split(':')[1].strip()
+                    raise MesonException(
+                        f'Error parsing "{str(configuration_file)}", option "{f.name}" '
+                        f'has invalid value: "{value}"')
```

Comment:
We go from:
```
ValueError: Not a boolean: true;

ERROR: Unhandled python exception
```

to

```
ERROR: Error parsing "meson.format", option "wide_colon" has invalid value: "true;"
```

Aside for the redundancy, I was struck by the fact that configparser stated the issue was it isn't a valid boolean, but we have thrown away that information.

Reply:
Okay, that makes sense. I've updated to provide more information about which file and key, but used the error message from the ValueError